lib/mappers/michigan_mapper.py

```diff
@@ -307,4 +318,17 @@ def map_multiple_fields(self):
         self.map_collection()
         self.map_date_and_publisher()
         self.map_data_provider_and_intermediate_provider()
-        self.map_object_and_is_shown_at()
\ No newline at end of file
+        self.map_object_and_is_shown_at()
+
+    # Get values from property
+    # Returns list of values
+    def get_date(self, s):
+        dates = []
+
+        if not isinstance(getprop(s, "dateIssued"), basestring):
+            for d in iterify(getprop(s, "dateIssued")):
+                dates.append(getprop(d, "#text"))
```

Comment:
I guess, but it will do the same thing, with the same risk of error (undefined `#text` key) as the existing code, with the advantage of also working if you encounter a string... not urgent but low risk I think. Up to you as to whether to change it!